test/conftest.py

```diff
@@ -1,5 +1,16 @@
+import platform
 import sys
 
+import pytest
+
 # We support Python 3.6+ for async code
 if sys.version_info[:2] < (3, 6):
     collect_ignore_glob = ["async/*.py", "with_dummyserver/async/*.py"]
+
+# The Python 3.8+ default loop on Windows breaks Tornado
+@pytest.fixture(scope="session", autouse=True)
+def configure_windows_event_loop():
+    if sys.version_info >= (3, 8) and platform.system() == "Windows":
+        import asyncio
+
+        asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
This is going to be a problem at some point – hip itself will need to support, and be tested on, the default event loop :-/. (Not something for this PR to worry about of course.)

Reply:
Thanks, opened https://github.com/python-trio/hip/issues/154 so that we don't forget about it